storage/postgres.go

```diff
@@ -1,2 +1,28 @@
 // Package storage provides postgres database configuration for storing messages.
 package storage
+
+import (
+	"database/sql"
+	"fmt"
+
+	_ "github.com/lib/pq"
+)
+
+const (
+	connFmt = "host=%v port=%v user=%v password=%v dbname=%v sslmode=disable"
+)
+
+func Connect(psqlHost, psqlPort, psqlUser, psqlPass, psqlDB string) (*sql.DB, error) {
```

Comment:
Pack paramaters to a structure please.

Reply:
packed to PostgresConfig and moved to config package